src/components/Meta.tsx

```diff
@@ -37,29 +45,23 @@ export default function render(
           "docs sectionType-" + props.sectionType + " " + props.category
         }
       />
+      {/* SEO content */}
+      <meta name="description" content={props.fields?.seoDescription}/>
```

Comment:
Why not use props.highlights as well here as we did before? I think this is needed for the historical MD documents, please investigate @zachserviss